### PR TITLE
Update exchange.graphql and add confirmPickup mutation

### PR DESCRIPTION
**Pull request by ashkan18**
# Change
`confirmPickup` mutation was added in https://github.com/artsy/exchange/pull/201 This PR catches up with that and adds `confirmPickup` mutation to MP

Depends on https://github.com/artsy/exchange/pull/201

### CODE REVIEW

File: _schema.graphql [hunk 1]
```diff
@@ -1032,7 +1032,7 @@ type ArtworkContextFair {
     section: String
 
     # Sorts for shows in a fair
-    sort: ShowSort = FEATURED_DESC
+    sort: ShowSort = null
```
Comment (ashkan18) [inline _schema.graphql]: not sure why these are here

Comment (orta) [reply _schema.graphql]: This is something to do with schema stitching - it's blocking me on 
https://github.com/artsy/metaphysics/pull/1311 too

Comment (orta) [reply _schema.graphql]: I'll fix it today and unblock this PR

**Comment by ashkan18** [comment]
@orta i was expecting to see `confirmPickup` mutation in the danger message ☝️ , right?

File: _schema.graphql [hunk 2]
```diff
@@ -4416,6 +4427,9 @@ type Mutation {
   # Fulfills an Order with one fulfillment by setting this fulfillment to all line items of this order
   fulfillOrderAtOnce(input: FulfillOrderAtOnceInput!): FulfillOrderAtOncePayload
 
+  # Confrims pickup for an ecommerce order
+  confirmPickup(input: ConfirmPickupInput!): ConfirmPickupPayload
```
Comment (orta) [inline _schema.graphql]: Please drop this one, I know it'll be inconsistent in clients (and I'd like to go and migrate everything to use `ecommerce` asap)

**Comment by orta** [comment]
Thanks you

**Pull request merged**